tests/ttnn/unit_tests/operations/test_conv_transpose2d.py

```diff
@@ -89,6 +90,11 @@ def run_conv_transpose2d(
     tt_weight_tensor = ttnn.from_torch(
         torch_weight_tensor, weights_dtype if weights_dtype != ttnn.bfloat8_b else ttnn.float32
     )
+    if not mirror_kernel:
+        torch_flipped_weights = torch.flip(torch_weight_tensor, [2, 3])
```

Comment:
should this be flipped in the weight pre-processing instead of in the test?

Reply:
If `mirror_kernel` is True, then the weights are flipped in the pre-processing code.

If it's set to False, the pre-processing doesn't flip the weights. In that case, the weights have to be flipped in the test code.